lib/galaxy/config/__init__.py

```diff
@@ -48,12 +48,12 @@
     get_stack_facts,
     register_postfork_function
 )
+from galaxy.webapps.galaxy import GALAXY_CONFIG_SCHEMA_PATH
```

Comment:
galaxy-webapps imports galaxy-app - so I think we need to move the schema not do this.

Reply:
The schema file is only referenced in `config/__init__.py` and `config/config_manage.py`; and, I think, those are the only two places it should be referenced from. So, considering a schema file is integral to `galaxy`, +1 for moving it (into `galaxy`, I assume). (@martenson - related to what we discussed..)